packages/database/src/migrations/20210920050147-NetEnrolmentRatesMapOverlayLaos-modifies-data.js

```diff
@@ -0,0 +1,256 @@
+    mapOverlayGroupId: DISTRICT_OVERLAY_GROUP.id,
+    sortOrder: 1,
+  },
+  {
+    dataElement: 'ner_district_use_t',
+    name: 'NER Upper Secondary',
+    reportCode: 'LESMIS_ner_upper_secondary_district',
+    mapOverlayGroupId: DISTRICT_OVERLAY_GROUP.id,
+    sortOrder: 2,
+  },
+  {
+    dataElement: 'ner_province_pe_t',
+    name: 'NER Primary',
+    reportCode: 'LESMIS_ner_primary_province',
+    mapOverlayGroupId: PROVINCE_OVERLAY_GROUP.id,
+    sortOrder: 0,
+  },
+  {
+    dataElement: 'ner_province_lse_t',
+    name: 'NER Lower Secondary',
+    reportCode: 'LESMIS_ner_lower_secondary_province',
+    mapOverlayGroupId: PROVINCE_OVERLAY_GROUP.id,
+    sortOrder: 1,
+  },
+  {
+    dataElement: 'ner_province_use_t',
+    name: 'NER Upper Secondary',
+    reportCode: 'LESMIS_ner_upuer_secondary_province',
+    mapOverlayGroupId: PROVINCE_OVERLAY_GROUP.id,
+    sortOrder: 2,
+  },
+];
+
+const getMapOverlayGroupId = async function (db, code) {
+  const results = await db.runSql(`select id from map_overlay_group where code = '${code}';`);
+
+  if (results.rows.length > 0) {
+    return results.rows[0].id;
+  }
+
+  throw new Error('MapOverlayGroup not found');
+};
+
+const addMapOverlayGroupRelation = async (db, parentId, childId, childType = 'mapOverlayGroup') => {
+  return insertObject(db, 'map_overlay_group_relation', {
+    id: generateId(),
+    map_overlay_group_id: parentId,
+    child_id: childId,
+    child_type: childType,
+  });
+};
+
+const getReport = (reportCode, dataElement) => ({
+  id: generateId(),
+  code: reportCode,
+  config: {
+    fetch: {
+      dataElements: [dataElement],
+      aggregations: [
+        {
+          type: 'FINAL_EACH_YEAR',
+          config: {
+            dataSourceEntityType: reportCode.includes('province') ? 'district' : 'sub_district',
+            aggregationEntityType: 'requested',
+          },
+        },
+      ],
+    },
+    transform: [
+      {
+        transform: 'updateColumns',
+        insert: {
+          organisationUnitCode: '=$organisationUnit',
+          value: '=divide($value, 100)',
+        },
+        exclude: ['organisationUnit', 'dataElement', 'period'],
+      },
+    ],
+  },
+});
+
+const getMapOverlay = (name, reportCode) => ({
+  id: reportCode,
+  name,
+  userGroup: PERMISSION_GROUP,
+  dataElementCode: 'value',
+  isDataRegional: true,
+  measureBuilder: 'useReportServer',
+  measureBuilderConfig: {
+    dataSourceType: 'custom',
+    reportCode,
+  },
+  presentationOptions: {
+    scaleType: 'performance',
+    displayType: 'shaded-spectrum',
+    measureLevel: reportCode.includes('province') ? 'District' : 'SubDistrict',
+    valueType: 'percentage',
+    scaleBounds: {
+      left: {
+        max: 'auto',
+      },
+      right: {
+        min: 'auto',
+      },
+    },
+    periodGranularity: 'one_year_at_a_time',
+  },
+  countryCodes: '{"LA"}',
+  projectCodes: '{laos_schools}',
+});
+
+const getPermissionGroupId = async (db, name) => {
```

Comment:
For future reference, I have added a new util in `src/utilities` called `nameToId` that could be used in place of this function to clean up the file a bit.